pipelines/disable-resource-locks.yaml

```diff
@@ -4,6 +4,24 @@ trigger: none
 # Don't run against PRs
 pr: none
  
+parameters:
+  - name: subscriptions
+    type: string
+    values:
+    - 'DCD-CNP-PROD'
+    - 'DCD-CNP-DEV'
+    - 'DTS-SHAREDSERVICES-STG'
+    - 'DTS-SHAREDSERVICES-PROD'
+
+variables:
```

Comment:
this looks unused?

Reply:
The parameter is used to select the correct service connection. For some reason the name of the service connection for the CNP subs doesn't match the sub name. The person running the pipeline will select the sub and this will be mapped to the service connection listed in the corresponding variable.

Reply:
I think disable-resource-locks.yaml is not correct, it is missing service connection reference

Reply:
The service connection is selected automatically based on the subscription selected by the user. The azureSubscription input references the variable. I ran the logic on a test pipeline and it seemed to work.